SimPEG/directives/directives.py

```diff
@@ -2205,66 +2205,224 @@ def JtJv(v):
+    def threshold_value(self, value):
+        self._threshold_value = validate_float("threshold_value", value, min_val=0.0)
 
-    @threshold.setter
-    def threshold(self, value):
-        self._threshold = validate_float("threshold", value, min_val=0.0)
+    deprecate_property(
+        threshold_value, "threshold", "threshold_value", removal_version="0.19.0"
+    )
 
     @property
-    def normalization(self):
-        """Whether to normalize by the smallest sensitivity weight.
+    def threshold_method(self):
+        """Threshold method for how `threshold_value` is applied:
+
+            - **global:** `threshold_value` is added to the cell weights prior to normalization; must be greater than 0.
+            - **percentile:** the smallest root-mean squared sensitivity is set using percentile threshold; must be between 0 and 100
+            - **amplitude:** the smallest root-mean squared sensitivity is a fractional percent of the largest value; must be between 0 and 1
+
 
         Returns
         -------
-        bool
+        str
         """
-        return self._normalization
+        return self._threshold_method
 
-    @normalization.setter
-    def normalization(self, value):
-        self._normalization = validate_type("normalization", value, bool)
+    @threshold_method.setter
+    def threshold_method(self, value):
+        self._threshold_method = validate_string(
+            "threshold_method", value, string_list=["global", "percentile", "amplitude"]
+        ).lower()
 
-    def initialize(self):
-        """
-        Calculate and update sensitivity
-        for optimization and regularization
+    @property
+    def normalization_method(self):
+        """Normalization method applied to sensitivity weights.
+
+        Options are:
+
+            - ``None``: normalization is not applied
+            - **maximum:** sensitivity weights are normalized by the largest value such that the largest weight is equal to 1.
+            - **minimum:** sensitivity weights are normalized by the smallest value, after thresholding, such that the smallest weights are equal to 1.
+
+        Returns
+        -------
+        None, str
         """
+        return self._normalization_method
+
+    @normalization_method.setter
+    def normalization_method(self, value):
+
+        if value is None:
+            self._normalization_method = value
+
+        elif isinstance(value, bool):
+            warnings.warn(
+                "Boolean type for 'normalization_method' is deprecated and will be removed in 0.19.0."
+                "Please use None, 'maximum' or 'minimum'."
+            )
+            if value:
+                self._normalization_method = "maximum"
+            else:
+                self._normalization_method = None
+
+        else:
+            self._normalization_method = validate_string(
+                "normalization_method", value, string_list=["minimum", "maximum"]
+            ).lower()
+
+    deprecate_property(
```

Comment:
Same comment as before, you need to store this on the class as the old name,
`normalization = deprecated_property(...)`